### PR TITLE
InvokeStatic reaper logMethodEntry to avoid unnecesary GETSTATIC instruction

### PR DESCRIPTION
**Pull request by rbro112**

### CODE REVIEW

**Comment by rbro112** [comment]
* **#199** <a href="https://app.graphite.dev/github/pr/EmergeTools/emerge-android/199?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rbro112 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/instrumentation/reaper/ReaperClassLoadTransform.kt [hunk 1]
```diff
@@ -103,23 +103,15 @@ class ReaperClassLoadMethodVisitor(
   override fun visitCode() {
```
Comment (rbro112) [inline gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/instrumentation/reaper/ReaperClassLoadTransform.kt:103]: I renamed this file to `ReaperClassLoadTransform` to match `SnapshotsPreviewRuntimeRetentionTransform`, as I was trying to search for the file and `ReaperClassLoad` managed to be tougher to find than I wanted.

Comment (chromy) [reply gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/instrumentation/reaper/ReaperClassLoadTransform.kt:103]: sg!

**Review by chromy — APPROVED** [review]
nice! lgtm

**Pull request merged**